spring-batch-docs/asciidoc/step.adoc

```diff
@@ -1424,59 +1428,58 @@ So far, all of the job configurations discussed have had at least
 ----
 
 If no transitions are defined for a `Step`,
-      then the `Job`'s statuses will be defined as
+      then the `Job`'s statuses is defined as
       follows:
 
 
 * If the `Step` ends with
           `ExitStatus` FAILED, then the
-          `Job`'s `BatchStatus` and
-          `ExitStatus` will both be FAILED.
-
+          `BatchStatus` and
+          `ExitStatus` of the `Job` are both `FAILED`.
 
-* Otherwise, the `Job`'s
+* Otherwise, the
           `BatchStatus` and
-          `ExitStatus` will both be COMPLETED.
+          `ExitStatus` of the `Job` are both `COMPLETED`.
 
 While this method of terminating a batch job is sufficient for
       some batch jobs, such as a simple sequential step job, custom defined
       job-stopping scenarios may be required. For this purpose, Spring Batch
       provides three transition elements to stop a `Job`
-      (in addition to the <<step.adoc#nextElement,"next" element>>
-      that we discussed previously). Each of these stopping elements will stop
+      (in addition to the <<step.adoc#nextElement,`next` element>>
+      that we discussed previously). Each of these stopping elements stops
       a `Job` with a particular
       `BatchStatus`. It is important to note that the
-      stop transition elements will have no effect on either the
+      stop transition elements have no effect on either the
       `BatchStatus` or `ExitStatus`
-      of any `Steps` in the `Job`:
-      these elements will only affect the final statuses of the
+      of any `Steps` in the `Job`.
+      These elements affect only the final statuses of the
       `Job`. For example, it is possible for every step
-      in a job to have a status of FAILED but the job to have a status of
-      COMPLETED, or vise versa.
+      in a job to have a status of `FAILED` but for the job to have a status of
+      `COMPLETED`. (It is also possible for the steps to have a status of `FAILED` and the job to have a status of `COMPLETED`.)
```

Comment:
I don't understand the sentence in parens.  Isn't that echoing what the previous sentence just said?

Reply:
Yes. Remove that sentence, please. I recall that I was making sure every such place had a sentence like that, and I seem to have gotten carried away. Thanks.